sktime/utils/_testing/estimator_checks.py

```diff
@@ -426,3 +429,31 @@ def check_persistence_via_pickle(Estimator):
     for method in results:
         unpickled_result = getattr(unpickled_estimator, method)(*args[method])
         _assert_almost_equal(results[method], unpickled_result)
+
+
+def check_multiprocessing_determinism(Estimator):
+    if 'n_jobs' in signature(Estimator.__init__).parameters:
+        estimator = _construct_instance(Estimator)
+        fit_args = _make_args(estimator, "fit")
+        ops = ["predict", "predict_proba", "transform"]
+        for op in ops:
+            if hasattr(estimator, op):
+                X_test = _make_args(estimator, op)[0]
+                result_set = []
+                for n_jobs in [1, 4]:
+                    estimator = _construct_instance(Estimator, n_jobs=n_jobs)
+                    if hasattr(estimator, 'n_jobs'):
+                        assert estimator.n_jobs == n_jobs
+                    set_random_state(estimator)
+                    estimator.fit(*fit_args)
+                    result_set.append(getattr(estimator, op)(X_test))
+
+                if isinstance(result_set[0], pd.DataFrame):
+                    assert_frame_equal(result_set[0], result_set[1])
+                else:
+                    np.testing.assert_array_almost_equal(
```

Comment:
should that be `assert_array_equal` in line with `assert_frame_equal`?

Reply:
Yes, you are right. The arrays should be just equal.